### PR TITLE
Angle Returns!

### PR DESCRIPTION
**Pull request by DangerRevolution**
## About the PR

- Readded Angle
- Did a general lighting pass to cut the lights on the map
- Did a decal, decoration and roleplay space pass (thanks to Noctrn)
- Moved Salvage into Logistics
- Moved Robotics to Epistemics
## Why / Balance

I like Angle, Colin didn't want to do it due to the department moving required, so I remapped it.
## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

Refer to the map render :O

**Changelog**
:cl: DangerRevolution
- add: Angle returns!

### CODE REVIEW

**Comment by deltanedas** [comment]
love when map render decides to not work when making a major map change :trollface:

**Comment by DebugOk** [comment]
> love when map render decides to not work when making a major map change :trollface:

Because this pr is lacking a valid station prototype

**Review by DebugOk — CHANGES_REQUESTED** [review]
This is missing everything required to function as a game map, converting to draft so actions usage isn't used on this

**Comment by VMSolidus** [comment]
You also need to touch these files for it to be a valid map:
image

image

image

**Comment by DangerRevolution** [comment]
> You also need to touch these files for it to be a valid map: image
> 
> image
> 
> image

waiting on velcro to help me w it on monday, plus discussing specific numbers and whatnot regarding player slots. i was new when playing on Angle so don't remember much abt the meta-aspects of it

**Comment by DangerRevolution** [comment]
just need to confirm with velcro which shuttle to use, forgot about that :O

**Comment by DangerRevolution** [comment]
not sure what it doesn't like about Line 2

File: Resources/Prototypes/Maps/angle.yml
Comment (IamVelcroboy) [inline Resources/Prototypes/Maps/angle.yml]: `-  type: gameMap`
Missing the `-`

File: Resources/Prototypes/Maps/angle.yml
Comment (IamVelcroboy) [inline Resources/Prototypes/Maps/angle.yml]: Empty shuttle path

File: Resources/Prototypes/Maps/Pools/default.yml
Comment (IamVelcroboy) [inline Resources/Prototypes/Maps/Pools/default.yml]: Keep it alphabetical. (Yes, Hive/Tortuga should be switched)

**Review by IamVelcroboy — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Maps/angle.yml
Comment (DangerRevolution) [reply Resources/Prototypes/Maps/angle.yml]: i'll wait for you to port over the nyano one

**Comment by DangerRevolution** [comment]
just gonna open for review to test the linter

File: Resources/Prototypes/Maps/angle.yml
Comment (IamVelcroboy) [reply Resources/Prototypes/Maps/angle.yml]: It's been ported

**Comment by IamVelcroboy** [comment]
Is this still WIP? Or ready for review?

**Comment by DangerRevolution** [comment]
> Is this still WIP? Or ready for review?

ready for review except i need your opinion on the uh job slots.

**Comment by Colin-Tel** [comment]
There's no station maps on the station.

I also think that the missing O2 gas miner should be replaced with an actual one, and the plasma miner should be removed.

**Comment by deltanedas** [comment]
like shoukou angle is meant to have no o2 miner, which meant that almost every angle round ended up with everyone dying of pressure and/or airloss

**Comment by DangerRevolution** [comment]
> There's no station maps on the station.
> 
> I also think that the missing O2 gas miner should be replaced with an actual one, and the plasma miner should be removed.

Will fix

**Comment by DangerRevolution** [comment]
> like shoukou angle is meant to have no o2 miner, which meant that almost every angle round ended up with everyone dying of pressure and/or airloss

Will fix

**Review by DebugOk — APPROVED** [review]
File changes look fine to me, job slots and the actual map will require Velcro's approval

**Comment by DebugOk** [comment]
@IamVelcroboy Have these issues been resolved yet?

**Comment by DangerRevolution** [comment]
it needs a minor lighting pass that i'm still making my way through

**Comment by IamVelcroboy** [comment]
> @IamVelcroboy Have these issues been resolved yet?

Review was done in Discord. Waiting for the changes requested there to be made.

**Comment by deltanedas** [comment]
@DangerRevolution is this real or fake

**Comment by DangerRevolution** [comment]
> @DangerRevolution is this real or fake

It's missing Shipyard and a few misc changes, but is 99% done last I was doing it. But it realistically needs a complete lighting overhaul and probably a ground-up remake as there's literally no room for Justice or any new rooms that are needed.

I mean, I could illegally expand the station using galvanised square steel and some screws from my aunt but then the Justice Department would look wack and out of place and not Angle-y

**Comment by deltanedas** [comment]
justice shittle that flies around independently :trollface: :trollface: :trollface:

**Comment by IamVelcroboy** [comment]
This is the last review I did on the map. If it is confirmed that the stuff in the review has been addressed, or it is addressed in another update, I will review again from there. [Discord Review](https://discord.com/channels/968983104247185448/1197421955926741042/1216806455022321677)

**Comment by DangerRevolution** [comment]
> This is the last review I did on the map. If it is confirmed that the stuff in the review has been addressed, or it is addressed in another update, I will review again from there. [Discord Review](https://discord.com/channels/968983104247185448/1197421955926741042/1216806455022321677)

I addressed 99% of the issues you brought up in the review; not sure if it's pushed to remote.

Waiting on #721 to be merged and the follow-ups before I get back to this.

**Comment by ghost** [comment]
The age old question... Security borgs has been merged; Did Danger ever get back to this? It's been such a hot minute this has been touched, and Fox picked up Angle in that time, to my knowledge

**Comment by DangerRevolution** [comment]
> The age old question... Security borgs has been merged; Did Danger ever get back to this? It's been such a hot minute this has been touched, and Fox picked up Angle in that time, to my knowledge

No; I haven't touched it, and have no plans to.

**Comment by TadJohnson00** [comment]
Noted, thanks for informing us.

**Pull request closed without merging**
